### DISCUSSION TITLE
How to add persistent storage?

### DISCUSSION THREAD

**Discussion by homa8511** [evroon/bracket#579] [Q&A] [answered]
Running bracket as docker images will lead to data loss each time it is stopped.

I want to set up persistent storage. I would add a folder for psql so that the database will store it's data to a folder.

But what about bracket itself? I want to store .env file and maybe uploaded files like team logos can be stored on file system.

How should a folder be mounted into bracket containers so that data will persist restarts?

**Comment by robigan** [comment, marked as answer]
See official postgres documentation. Additionally you'll have to mount /app/static in the backend to a volume to save images properly
